pandas/core/groupby.py

```diff
@@ -1055,6 +1056,60 @@ def tail(self, n=5):
         tail = obj[in_tail]
         return tail
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply func(self, \*args, \*\*kwargs)
+
+        .. versionadded:: 0.16.3
+
+        Parameters
+        ----------
+        func : function
+        function to apply to the %(klass)s.
+        ``args``, and ``kwargs`` are passed into ``func``.
+        Alternatively a ``(callable, data_keyword)`` tuple where
+        ``data_keyword`` is a string indicating the keyword of
+        ``callable`` that expects the %(klass)s.
+        args : positional arguments passed into ``func``.
+        kwargs : a dictionary of keyword arguments passed into ``func``.
+
+        Returns
+        -------
+        object : the return type of ``func``.
+
+        Notes
+        -----
+
+        Use ``.pipe`` when chaining together functions that expect
+        a GroupBy, or when alternating between functions that take
+        a DataFrame and a GroupBy.
+
+        Assuming that one has a function f that takes and returns
+        a DataFrameGroupBy, a function g that takes a DataFrameGroupBy
+        and returns a DataFrame, and a function h that takes a DataFrame,
+        instead of having to write:
+
+        >>> f(g(h(df.groupby('group')), arg1=a), arg2=b, arg3=c)
+
+        You can write
+
+        >>> (df
+        ...    .groupby('group')
+        ...    .pipe(f, arg1)
+        ...    .pipe(g, arg2)
+        ...    .pipe(h, arg3))
+
+
+        See Also
+        --------
+        pandas.NDFrame.pipe
```

Comment:
Can you make this `pandas.Series.pipe` and `pandas.DataFrame.pipe` (we don't have docs of NDFrame)